flink/json-jobs.yml

```diff
@@ -0,0 +1,119 @@
+type: json
+data:
+  url: /v1/jobs/metrics?get=numberOfFailedCheckpoints,lastCheckpointSize,lastCheckpointExternalPath,totalNumberOfCheckpoints,lastCheckpointRestoreTimestamp,lastCheckpointAlignmentBuffered,restartingTime,uptime,numberOfInProgressCheckpoints,downtime,numberOfCompletedCheckpoints,fullRestarts,lastCheckpointDuration
+  server: ${FLINK_HOSTPORT}
+
+observation:
+  - name: jobs_uptime
+    metricNamespace: flink
+    path: $.[?(@.id=uptime)]
+    metric:
+      - name: jobs.uptime
+        source: sum
+        type: gauge
+        label: uptime
+        description: The time that the job has been running without interruption
+
+  - name: jobs_downtime
+    metricNamespace: flink
+    path: $.[?(@.id=downtime)]
+    metric:
+      - name: jobs.downtime
+        source: sum
+        type: gauge
+        label: downtime
+        description: "For jobs currently in a failing/recovering situation, the time elapsed during this outage. 0 for running jobs and -1 for completed jobs"
+
+  - name: jobs_restartingtime
+    metricNamespace: flink
+    path: $.[?(@.id=restartingTime)]
+    metric:
+      - name: jobs.restartingtime
+        source: sum
+        type: gauge
+        unit: ms
+        label: restarting time
+        description: The time it took to restart the job, or how long the current restart has been in progress
+
+  - name: jobs_full_restarts
+    metricNamespace: flink
+    path: $.[?(@.id=fullRestarts)]
+    metric:
+      - name: jobs.full.restarts
+        source: sum
+        type: gauge
```

Comment:
Gauge or count? Similar for others below.

Reply:
counter